web-client/src/presenter/actions/validatePetitionerAction.ts

```diff
@@ -26,11 +25,9 @@ export const validatePetitionerAction = ({
       existingPetitioners: caseDetail.petitioners,
     });
 
-  store.set(state.validationErrors.contact, errors);
-
   if (isEmpty(errors)) {
     return path.success();
   } else {
-    return path.error({ errors });
+    return path.error({ errors: { contact: errors } });
```

Comment:
The way this action was functioning was failing to update the corresponding form (`EditPetitionerInformationInternal.tsx`) appropriately.